cmd/crc/cmd/start.go

```diff
@@ -18,6 +27,30 @@ var startCmd = &cobra.Command{
 	},
 }
 
+var (
+	bundlePath string
+	vmDriver   string
+)
+
 func runStart(arguments []string) {
+
+	// TODO: this should be a validation
+	if vmDriver != "libvirt" {
+		errors.ExitWithMessage(1, "Unsupported driver: %s", vmDriver)
+	}
+
 	preflight.StartPreflightChecks()
+
+	startConfig := machine.StartConfig {
+		Name:       defaultName,
+		BundlePath: bundlePath,
+		VMDriver:   vmDriver,
+		Debug:      false,
```

Comment:
This debug is to get the machine logs, so are we going to add the logic about how to enable it as of now this is just `false`?

Reply:
at this point this is only helpful to us, until we have modified the output in libmachine. often their errors are misleading as h*ll and include wrong suggestions, such as running `docker-machine create` when no VM exists, or other stuff for ssh, while VM did not get an IP.